src/main.ts

```diff
@@ -17,18 +17,33 @@ async function run(): Promise<void> {
     const octokit = github.getOctokit(inputs.token);
 
     const ownership = {
-      owner: github.context.repo.owner,
-      repo: github.context.repo.repo,
+      owner: '',
```

Comment:
Could you keep the default values the same? It makes it more explicit what will be the default and how/when it is overridden.

Reply:
Right, not sure I complicated it like this. Fixed :)